app/controllers/send_application_controller.rb

```diff
@@ -14,7 +15,35 @@ def update
     end
   end
 
+  private
+
   def send_email
-    # send email
+    ApplicationMailer.snap_application_notification(
+      file_name: new_file_name,
+      recipient_email: current_snap_application.email,
+    ).deliver
+  end
+
+  def create_dhs1171_pdf
+    data = {
+      applying_for_food_assistance: "Yes",
+      full_name: current_snap_application.name,
+      birth_day: current_snap_application.birthday.strftime("%d"),
+      birth_month: current_snap_application.birthday.strftime("%m"),
+      birth_year: current_snap_application.birthday.strftime("%Y"),
+      street_address: current_snap_application.street_address,
+      city: current_snap_application.city,
+      county: current_snap_application.county,
+      state: current_snap_application.state,
+      zip: current_snap_application.zip,
+      signature: current_snap_application.signature,
+      signature_date: current_snap_application.signed_at,
+    }
```

Comment:
I wonder if this data translation/adapter hash could move into the `Dhs1171Pdf` class instead of here in the controller...